x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.test.tsx

```diff
@@ -291,37 +279,6 @@ describe('Alert table context menu', () => {
           expect(button).toBeDisabled();
         });
       });
-
-      describe("when users don't have write event filters privilege", () => {
```

Comment:
removed this but will try to implement some other test in place of the old one, we always have actions available now and it seems that this logic depended on action count == 0 to disable the button.